### PR TITLE
[local-preview] Add information about time for gitpod ready

### PR DESCRIPTION
**Pull request by Pothulapati**
## Description

This PR adds caution for users to wait before exiting Gitpod, as it
takes around 5 minutes to be ready usually. This surfaces that
information to the user, without thinking that something is wrong

Signed-off-by: Tarun Pothulapati <tarun@gitpod.io>

## Related Issue(s)

Fixes #

## Release Notes

```release-note
[local-preview] Add information about time for gitpod ready
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2618"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2618"></a>

### CODE REVIEW

File: gitpod/docs/self-hosted/latest/local-preview.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Unpacking the above command:
 
 ## 2. Accessing Gitpod
 
+> **Note:** It usually takes 5 minutes to go from `starting Gitpod` status to `Gitpod is ready`. Following is how it looks:
```
Comment (Pothulapati) [inline gitpod/docs/self-hosted/latest/local-preview.md:42]: I lost my changes as I unexpectedly deleted the file! Sorry :/

Can you rephrase it again? @lucasvaltl

**Review by lucasvaltl — APPROVED** [review]
LGTM :)
